adafruit_magtag/magtag.py

```diff
@@ -300,9 +300,10 @@ def set_text(self, val, index=0, auto_refresh=True):
         if auto_refresh:
             self.refresh()
 
-    def enter_deep_sleep(self, sleep_time):
+    def enter_deep_sleep_and_restart(self, sleep_time):
```

Comment:
it does the equivalent of `sys.exit()` _before_ going into deep sleep, so `exit_and_enter_deep_sleep()` denotes the order of what happens more exactly.

It would also be good to repeat or paraphrase the caveats in the `alarm` documentation about not actually entering full light or deep sleep if you are connected to USB. The sleeps are simulated when connected, so that you can debug your program, but it means power saving may be significantly curtailed.

Reply:
Ok, fair enough.